react-flexr/react-flexr.d.ts

```diff
@@ -0,0 +1,152 @@
+// Type definitions for react-flexr v2.0.2
+// Project: https://github.com/kodyl/react-flexr
+// Definitions by: Jeffery Grajkowski <https://github.com/pushplay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../react/react.d.ts" />
+
+declare namespace __ReactFlexr {
+    export import React = __React;
+
+    interface GridProps extends React.Props<Grid> {
+        /**
+         * Vertical Align Sub Cells: top, center, bottom
+         */
+        align?: "top" | "center" | "bottom";
+
+        /**
+         * Horizontal Align Sub Cells: left, center, right
+         */
+        hAlign?: "left" | "center" | "right";
+
+        /**
+         * Override default gutter: '1em', '5%', '10px', etc.
+         * Propagates downwards. Cells inside this Grid component
+         * will use the same gutter.
+         */
+        gutter?: string;
+
+        /**
+         * All sub cells will be full height.
+         */
+        flexCells?: boolean;
+    }
+
+    export class Grid extends React.Component<GridProps, {}> {
+    }
+
+    interface CellProps extends React.Props<Cell> {
+        /**
+         * Vertical Align This Cell: top, center, bottom
+         */
+        align?: "top" | "center" | "bottom";
+
+        /**
+         * Override default gutter: '1em', '5%', '10px', etc.
+         */
+        gutter?: string;
+
+        /**
+         * Cell will be full height.
+         */
+        flex?: boolean;
+
+        /**
+         * Takes a fraction. e.g. 1/6
+         */
+        size?: string | number;
+
+        /**
+         * Like size, but only for palm devices.
+         * Accepts 'hidden' as well.
+         */
+        palm?: string | number;
```

Comment:
That's not quite right because `like size` it can accept other strings.  eg: '1/6'.